.github/workflows/build_container.yml

```diff
@@ -0,0 +1,46 @@
+name: build container
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  # build amd64 images
+  jobs-amd64:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Login to Quay.io Registry
+      run: docker login -u=${{ secrets.DOCKERHUB_USERNAME }} -p=${{ secrets.DOCKERHUB_PASSWORD }} quay.io
+
+    - name: build amd64 image
+      run: ./build_container/docker_build.sh `date +'%Y-%m-%d'`
+
+    - name: create manifest
+      run: |
+        ./build_container/docker_manifest.sh cilium-llvm `date +'%Y-%m-%d'`
+        ./build_container/docker_manifest.sh cilium-iproute2 `date +'%Y-%m-%d'`
+        ./build_container/docker_manifest.sh cilium-bpftool `date +'%Y-%m-%d'`
+
+
+  # build arm64 images
+  jobs-arm64:
+    continue-on-error: true
+    runs-on: [self-hosted, Linux, ARM64]
```

Comment:
Nice, didn't realise these were supported. What is availability like?

Reply:
Or will this need to be a custom runner?

Reply:
yes,  we need to prepare arm64 environment,  Self-hosted runners can be physical, virtual, in a container, or in a cloud,
The machine can communicate with GitHub Actions;
Add runner  from  Settings->actions->add runner   page
Please refer to https://github.blog/2019-11-05-self-hosted-runners-for-github-actions-is-now-in-beta/

Reply:
I see, and have you tried using binfmt and qemu?

Here is what I did earlier: https://github.com/errordeveloper/buildx-test/blob/3e1ca27f81688f43bd4e6dfe420494ea4e5a81c1/.github/workflows/images.yaml#L12-L16

Reply:
I have tried to compile cilium image using Qemu on a bare-metal machine, it is worked,
Have not tried the "GH action + binfmt" method;
Does the community prefer to use "GitHub-hosted + binfmt" to build the multi-arch image

Reply:
Yeah, there is currently some infra that is sponsored and managed by Isovalent, and the aim is to move as much as possible to GitHub, so it's maintained as part of the project.
Having to host a custom GitHub Action runner would introduce a new component that someone would have to maintain, it's best to avoid that. Of course, test clusters are going to be hosted in AWS, GCP and other providers, but image builds shouldn't depend on anything outside GitHub.

Reply:
Make sense, I will update this PR again recently.